adapters/clients/replication.go

```diff
@@ -40,6 +41,88 @@ func NewReplicationClient(httpClient *http.Client) replica.Client {
 	}
 }
 
+func (c *replicationClient) FindObject(ctx context.Context,
+	host, index, shard string, id strfmt.UUID,
+) (*replica.ReadResponse[*storobj.Object], error) {
+	resp := &replica.ReadResponse[*storobj.Object]{}
+	req, err := newHttpReplicaRequest(ctx, http.MethodGet, host, index, shard, "", id.String(), nil)
+	if err != nil {
+		return nil, fmt.Errorf("create http request: %w", err)
+	}
+	err = c.do(c.timeoutUnit*90, req, nil, resp)
```

Comment:
`c.do()` assumes "resp" can be marshalled using standard json marshaler. However "resp" contains *storobj.Object" which has a custom Marshaler

Reply:
got it. will refactor

Reply:
done